### DISCUSSION TITLE
Any way to turn off the displaying of plots in Pluto, and instead use the UI?

### DISCUSSION THREAD

**Discussion by VarLad** [fonsp/Pluto.jl#1004] [Questions about using Pluto] [answered]
I use gnuplot for plotting
Unfortunately, features like rotate and zoom aren't supported in Pluto, but are supported when I plot them normally from terminal (or VSCode for that point)
Is there any way I can just turn off the showing of plots, and instead make the plots appear in a UI like normal?
Preferably, I'd like to be able to do this for individual plots instead of all my plots

**Comment by fonsp** [comment, marked as answer]
You might want to ask this to GnuPlot.jl instead -- we don't know how GnuPlot works.

**Reply by VarLad** [reply]
Never mind
display(plot) works 
Accidentally found it！
